client/components/dashboard/dashboard-version-service.js

```diff
@@ -113,14 +113,21 @@ DashboardVersionService.prototype.verifyAndUpdateModel = function(dashboard) {
   // current.
   if (version == this.currentVersion) {
     if (!goog.isDef(dashboard.version)) {
+      goog.asserts.assert(!goog.isNull(version.version));
```

Comment:
This looks wrong. If I'm understanding it right, in this branch dashboard.version === undefined, so it cannot be null. If you replace the assignment with `dashboard.version = undefined`, does it work without a warning? I suspect the problem may be with the dashboard.version declared type and not the code here.